src/test/ui/issues/issue-72574-1.stderr

```diff
@@ -0,0 +1,25 @@
+error: `_x @` is not allowed in a tuple
+  --> $DIR/issue-72574-1.rs:4:14
+   |
+LL |         (_a, _x @ ..) => {}
+   |              ^^^^^^^ is only allowed in a slice
+   |
+help: replace with `..` or use a different valid pattern
+   |
+LL |         (_a, ..) => {}
+   |              ^^
```

Comment:
It'd be nice if we also gave a suggestion to be able to use the fields:

```
help: alternatively, bind each tuple field independently
   |
LL |         (_a, field1, field2) => {}
   |              ^^^^^^^^^^^^^^
```

Edit: right, at parse time you don't _know_ the size of the tuple. To give this suggestion accurately you'd need to delay the error to after resolve. Let's instead either use placeholders for now or a stand alone `help`. Note that `help` will come before the suggestion (always) so consider changing the wording appropriately.

Reply:
> Let's instead either use placeholders for now or a stand alone help

Can you expand on this?  I'm not sure that I completely follow the ask here.

Reply:
I ended up going with this in https://github.com/rust-lang/rust/pull/72677/commits/e73ac086c54084dced1916d062267be4058b257c.  I think that this addresses your request here?

```
error: `_x @` is not allowed in a tuple
 --> src/main.rs:4:14
  |
4 |         (_a, ref mut _x @ ..) => {}
  |              ^^^^^^^^^^^^^^^ this is only allowed in slice patterns
  |
help: either remove this and bind each tuple field independently
 --> src/main.rs:4:14
  |
4 |         (_a, ref mut _x @ ..) => {}
  |              ^^^^^^^^^^^^^^^
help: or if you don't need to use the contents of _x, discard the tuple's remaining fields
  |
4 |         (_a, ..) => {}
  |              ^^

error: aborting due to previous error
```

Reply:
I changed the local test code that I am working with to include the `ref mut` binding mode.  The tests did not change. The example above is slightly different than what you will see in the test files